runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -247,6 +247,12 @@ public void onChatMessage(ChatMessage event)
 				return;
 			}
 		}
+
+		if (config.screenshotPet() && (chatMessage.contains("You have a funny feeling like you're being followed") || chatMessage.contains("You feel something weird sneaking into your backpack") || chatMessage.contains("You have a funny feeling like you would have been followed")))
```

Comment:
You should keep the chat message contains check on their own lines to avoid a really long line. Alternatively, a switch statement on chatMessage would probably be better.

Also, should it screenshot on "You have a funny feeling like you would have been followed" since you don't receive anything? I guess it's nice to know that you would have potentially received something :p

Reply:
A switch is way better, idk why I didn't think of that. I feel like most people (at least myself) would like to see everytime they hit the pet droptable, and I very much don't want to add a toggle for it :p